tests/test_relations.py

```diff
@@ -176,6 +176,14 @@ def test_representation(self):
         representation = self.field.to_representation(self.instance)
         assert representation == self.instance.name
 
+    def test_no_queryset_init(self):
+        class NoQuerySetSlugRelatedField(serializers.SlugRelatedField):
+            def get_queryset(this):
```

Comment:
It perhaps would have been better to use `staticmethod`, or at least add a comment about why it was like that.